etc/rpm/gen-spec.sh

```diff
@@ -0,0 +1,54 @@
+cat <<-EOF
+%global __os_install_post /usr/lib/rpm/brp-compress %{nil}
+%global __spec_install_post %{nil}
+%global debug_package %{nil}
+
+Name:           tikv
+Version:        4.0.0.alpha
```

Comment:
can we get it from git command

Reply:
Yes! You're right! I need to update this script so we don't need to update this. :)